### ISSUE TITLE
Support of AFS (Apple File System)

### ISSUE THREAD

**Issue by tomkohen** [darlinghq/darling-dmg#39]
Would be happy to see someday support of new Apple FS: 
https://developer.apple.com/library/prerelease/content/documentation/FileManagement/Conceptual/APFS_Guide/Introduction/Introduction.html

**Comment by LubosD** [comment]
Yep, thought the same yesterday. Do you happen to know if AFS will also be used in DMGs? I assume yes, but just to be sure.

**Comment by tomkohen** [comment]
Sorry for the late reply. Nope, still haven't met AFS in DMG

**Comment by tomkoen** [comment]
Ok, now it's easy to create DMG with APFS:

hdiutil create -volname myVolume -srcfolder ~/MyVolumeFolder -ov -fs apfs myVolume.dmg

**Comment by jack980517** [comment]
The DMGs in IPSWs (iOS firmware files) uses APFS since iOS 10.3.

**Comment by tomkoen** [comment]
Just related info: https://bombich.com/blog/2018/02/15/macos-may-lose-data-on-apfs-formatted-disk-images

**Comment by tomkoen** [comment]
[Decoding the APFS file system](https://github.com/darlinghq/darling-dmg/files/1741393/DIIN_698_Revisedproof.pdf)

**Comment by tomkoen** [comment]
https://github.com/sgan81/apfs-fuse

**Comment by zivkaspersky** [comment]
as a default if you create a DMG in latest version of hdutil it create a GPT partitioned disk with a APFS file system.

**Comment by TheRedXD** [comment]
bump
